downstream/modules/platform/ref-ansible-inventory-variables.adoc

```diff
@@ -11,47 +11,47 @@ For a list of global configuration options, see link:https://docs.ansible.com/an
 [cols="50%,50%",options="header"]
 |====
 | *Variable* | *Description*
-| *`ansible_connection`* | The connection plugin used for the task on the target host.
+| `ansible_connection` | The connection plugin used for the task on the target host.
 
 This can be the name of any of Ansible connection plugins.
 SSH protocol types are `smart`, `ssh` or `paramiko`.
 
 Default = `smart`
-| *`ansible_host`* | The IP or name of the target host to use instead of *`inventory_hostname`*.
-| *`ansible_port`* | The connection port number.
+| `ansible_host` | The IP or name of the target host to use instead of `inventory_hostname`.
+| `ansible_port` | The connection port number.
 
-Default: 22 for ssh
-| *`ansible_user`* | The user name to use when connecting to the host.
-| *`ansible_password`* | The password to authenticate to the host.
+Default: 22 for SSH
+| `ansible_user` | The user name to use when connecting to the host.
+| `ansible_password` | The password to authenticate to the host.
 
 Never store this variable in plain text.
 
 Always use a vault.
-| *`ansible_ssh_private_key_file`* | Private key file used by SSH. Useful if using multiple keys and you do not want to use an SSH agent.
+| `ansible_ssh_private_key_file` | Private key file used by SSH. Useful if using multiple keys and you do not want to use an SSH agent.
```

Comment:
```suggestion
| `ansible_ssh_private_key_file` | Private key file used by SSH. 
```
Duplicated sentence

Reply:
Good catch - it looks like this variable is listed twice !